app/blog.yaml

```diff
@@ -1,4 +1,9 @@
 ---
+- title:  "Upcoming changes to shadow DOM v0 and HTML imports"
```

Comment:
This gets awkwardly truncated from 

>"Upcoming changes to shadow DOM v0 and HTML imports"

to 

>"Upcoming changes to shadow DOM v0 and HTML imp..."

An option for a shorter title could be:

>"Shadow DOM v0 and HTML imports are changing"